terra-env/main.tf

```diff
@@ -72,7 +72,7 @@ module "sam" {
 }
 
 module "workspace_manager" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=terra-workspace-manager-0.6.5"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=jaycarlton%2fPF-551"
```

Comment:
Terraform plan LG. Just remember bump this version to terra-workspace-manager-0.6.6 before merge.